Flashcards.kronyer/Flashcards/DataAccess.cs

```diff
@@ -0,0 +1,322 @@
+﻿using Dapper;
+using Flashcards.Models;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Configuration;
+using Stack = Flashcards.Models.Stack;
+
+public class DataAcess
+{
+    IConfiguration configuration = new ConfigurationBuilder().AddJsonFile("appsettings.json").Build();
+
+    private string ConnectionString;
+
+    public DataAcess()
+    {
+        ConnectionString = configuration.GetSection("ConnectionStrings")["DefaultConnection"];
+    }
+
+    public void CreateTables()
```

Comment:
As a little fun challenge, see how you can create the database when the user runs the application rather than manually? `Note: this is not absolutely necessary but just a challenge. Not doing this will not affect your project's approval. If it is too difficult you can go ahead and skip it 😃 `